packages/react-devtools-shared/src/__tests__/legacy/inspectElement-test.js

```diff
@@ -873,7 +873,6 @@ describe('InspectedElementContext', () => {
         xyz: 1,
       },
     });
-    // $FlowFixMe
     const bigInt = BigInt(123); // eslint-disable-line no-undef
```

Comment:
And we should probably also start failing lint on unused suppressions because that is unneeded as of https://github.com/facebook/react/pull/26479